src/shipit_static_analysis/default.nix

```diff
@@ -13,6 +13,13 @@ let
   name = "mozilla-shipit-static-analysis";
   dirname = "shipit_static_analysis";
 
+  # Customize gecko environment with Nodejs & Python 3 for linters
```

Comment:
Does this mean we are going to use Python 3 for linting? We shouldn't do that, as it could cause us to warn about issues that are not issues for Python 2 (e.g. basestring)

Reply:
No it just needs to be also available in the environment for the `py3` linter.

Reply:
:+1: